text/0012-CLI-commands.md

```diff
@@ -425,6 +425,736 @@ qri get [flags] <dataset_reference> <dataset_reference2> ...
   -h, --help            help for get
 ```
 
+<a id='qri_info'></a>
+## qri info
+
+show summarized description of a dataset
+
+### Synopsis
+
+info describes datasets
```

Comment:
Can this be elaborated on, to clarify what particular info is shown? Is it exactly the same as what `list` shows (just for a single element) and if so I guess just say that?